app/controllers/general_feedback_controller.rb

```diff
@@ -7,14 +7,14 @@ def create
     @feedback = GeneralFeedback.new(general_feedback_params)
     return render 'new' unless @feedback.valid?
 
-    recaptcha_valid = verify_recaptcha(model: @feedback, action: 'feedback')
+    recaptcha_is_valid = verify_recaptcha(model: @feedback, action: 'feedback')
```

Comment:
`_is_valid` is discouraged in rails. `_valid?` is better

Reply:
`_valid?` is not permitted by Rubocop.